node/src/components/linear_chain_sync.rs

```diff
@@ -622,3 +708,45 @@ where
             move || Event::GetBlockHeightResult(block_height, BlockByHeightResult::Absent(cloned)),
         )
 }
+
+/// Returns key in the Global State under which the LinearChainSync's state is stored.
```

Comment:
Global state? Do you mean database?

Reply:
Done in [`6d6256a` (#904)](https://github.com/CasperLabs/casper-node/pull/904/commits/6d6256afbfb52e3c3bcc4ea817550a3ce25cea36).